v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java

```diff
@@ -326,6 +329,7 @@ public interface TokenizePipelineOptions extends DataflowPipelineOptions {
 
     @TemplateParameter.Integer(
         order = 4,
+        groupName = "Source",
```

Comment:
Same. This is about the DLP API call made in the middle. Probably makes sense to combine `inspectTemplateName` and `batchSize` under a separate 3rd group called "DLP Configuration".

Reply:
Done